staging/gatekeeper/README.md

```diff
@@ -5,7 +5,7 @@ Kubernetes.
 
 ## Prerequisites
 
-- Kubernetes 1.9 (or newer) for validating and mutating webhook admission
+- Kubernetes 1.19 (or newer) for validating and mutating webhook admission
```

Comment:
Is this a typo? Assuming this does work on older versions of Kubernetes?

Reply:
Good point! I checked and it seems they recommend 1.14 https://github.com/open-policy-agent/gatekeeper#minimum-kubernetes-version. I will update that